ansible_deployer/command_line.py

```diff
@@ -513,18 +561,16 @@ def main():
 
     start_ts = datetime.datetime.now().strftime("%Y%m%d_%H%M%S")
 
-    if len(sys.argv) < 2:
+    if len(sys.argv) < 3:
```

Comment:
This is wrong and is the reason why CI fails today. circle CI isn't related with the publication it calls pip on a local checkout

Reply:
think about ansible-deployer list (it's only two and it's fine)

Reply:
Yes, we need version 0.0.17. Commit is updated. Please upload the package I sent you on teams, credentials are required.